components/TokenBalanceCard.tsx

```diff
@@ -194,10 +199,27 @@ const TokenDeposit = ({
           // If the Proposal is in Voting state refetch it to make sure we have the latest state to avoid false positives
           proposal = await getProposal(connection, proposal.pubkey)
           if (proposal.info.state === ProposalState.Voting) {
-            // Note: It's technically possible to withdraw the vote here but I think it would be confusing and people would end up unconsciously withdrawing their votes
-            throw new Error(
-              `Can't withdraw tokens while Proposal ${proposal.info.name} is being voted on. Please withdraw your vote first`
+            const governance = await getGovernance(
```

Comment:
nit: There is no need to fetch governance from the chain here, you can use `{governances} = useRealm()` to get all the governances for the given realm